data/images/autotiles.satc

```diff
@@ -1,4818 +1,4997 @@
-;;
```

Comment:
I have no idea why the diff considered that I deleted the entire original file and replaced it with a new one instead of only adding my part. But it works the same